ArduPlane/Plane.h

```diff
@@ -925,7 +925,7 @@ class Plane : public AP_HAL::HAL::Callbacks {
     bool geofence_stickmixing(void);
     void geofence_send_status(mavlink_channel_t chan);
     bool geofence_breached(void);
-    void disarm_if_autoland_complete();
+    void disarm_if_autoland_complete(uint32_t timer);
```

Comment:
Can this just have a default argument value of 0?  It's making the stuff in AP_Landing look really confusing, or at least in need of a constant.